e2e/app/page/data-page.ts

```diff
@@ -38,27 +38,27 @@ export default class DataPage extends AuthenticatedPage {
     }
   }
 
-  async selectWorkspaceAction(action: WorkspaceAction) {
+  async selectWorkspaceAction(action: WorkspaceAction): Promise<void> {
     const ellipsisMenu = new EllipsisMenu(this.page, './/*[@data-test-id="workspace-menu-button"]');
-    await ellipsisMenu.selectAction(action);
+    return ellipsisMenu.selectAction(action);
   }
 
   /**
    * Select DATA, ANALYSIS or ABOUT page tab.
    * @param tabName
    */
-  async selectTab(tabName: PageTab): Promise<void> {
-    const selector = '//*[@aria-selected and @role="button"]';
-    await this.page.waitForXPath(selector, {visible: true});
-    const tabs = await this.page.$x(selector);
-    for (const tab of tabs) {
-      const contentProp = await tab.getProperty('textContent');
-      if (await contentProp.jsonValue() === tabName) {
-        return tab.click();
-      }
-      await tab.dispose();
-    }
-    throw new Error(`Failed to find page tab with name ${tabName}`);
+  async openTab(tabName: LabelAlias): Promise<void> {
```

Comment:
rename `selectTab` to `openTab` because I think `select` word can be confused with menu select.